tests/model_hub_tests/tensorflow/test_tf_convert_model.py

```diff
@@ -152,10 +152,10 @@ def clean_dir(self, dir_name: str):
 
     def teardown_method(self):
         # remove all downloaded files for TF Hub models
-        self.clean_dir(tf_hub_cache_dir)
+        # self.clean_dir(tf_hub_cache_dir)
 
         # remove all downloaded files for HF models
-        self.clean_dir(hf_cache_dir)
+        # self.clean_dir(hf_cache_dir)
```

Comment:
please do not leave commented out code. If this is in production state, so let us remove it.
Do we cache both TF Hub and HF models?

Reply:
I will remove them before the merge.
Yep, both directories were created on the shared drive. They reduce the execution time by an hour. It seems though that the cache doesn't work for pre-commit models scope.